<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addresscreen</class>
 <widget class="QDialog" name="Addresscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartShopXpress</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/img0/logo3.png</normaloff>:/img/img0/logo3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/img/img0/gbackground.png);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/address.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/menuicon.png);
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="addressCheckBox1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="addressCheckBox3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="addressCheckBox2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="addressCheckBox4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/lineEditBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addAddressPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/add.png);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterAddressNameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px</string>
   </property>
   <property name="text">
    <string>Address name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteAddressPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/img0/delete.png);
border-radius:25px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterAddressCityLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px</string>
   </property>
   <property name="text">
    <string>City</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterAddressStreetLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px</string>
   </property>
   <property name="text">
    <string>Street</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterAddressBuildingLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px</string>
   </property>
   <property name="text">
    <string>Building name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enterAddressDoorNumLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:20px</string>
   </property>
   <property name="text">
    <string>Door num</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
